README.md

```diff
@@ -15,10 +15,10 @@ CVAT annotations                             ---> Publication, statistics etc.
 ```
 <!--lint enable fenced-code-flag-->
 
-- [Getting started](https://openvinotoolkit.github.io/datumaro/docs/getting_started)
+- [Getting started](https://openvinotoolkit.github.io/datumaro/latest/docs/get-started/quick-start-guide)
 - [Features](#features)
-- [User manual](https://openvinotoolkit.github.io/datumaro/docs/user-manual)
-- [Developer manual](https://openvinotoolkit.github.io/datumaro/api)
```

Comment:
Would you add "Level Up" sections?

Reply:
Sure, reflected at https://github.com/JihwanEom/datumaro/commit/1a691ddc369e453d47c5d6009ea499ff757a8745